types/storage_key.go

```diff
@@ -91,6 +95,7 @@ func CreateStorageKey(meta *Metadata, prefix, method string, args ...[]byte) (St
 		return createKeyNMap(method, prefix, validatedArgs, entryMeta)
 	}
 
+	// Deprecated since metadata v14
 	if entryMeta.IsDoubleMap() {
```

Comment:
lets remove these methods and have just one method this is IsMap(bool)

Reply:
Done 👍